### DISCUSSION TITLE
0.14 beta - HomeAssistant API support for preview.gif

### DISCUSSION THREAD

**Discussion by Bod1985** [blakeblackshear/frigate#12142] [Ask A Question] [answered]
Hi,

Looking at the HTTP API docs I notice the new preview.gif option that is used presumably in the new PWA interface.

Is this also exposed to the HomeAssistant API?

The beta docs only list snapshot.jpg, latest.jpg and clip.mp4 in the Notifications API.

Cheers!

**Comment by NickM-27** [comment, marked as answer]
It will be in the next integration update

**Reply by Bod1985** [reply]
Thank you - I've been able to get this set up and in use.

The gif is very low quality though, and although it provides an okay overview of an event, it's inferior (on iOS in the context of notifications) to the android clip.mp4 which gets converted to a gif of still frames because of the resolution.

I would gladly use clip.mp4 on my iPhone but my upload speed is pretty bad so it really struggles to load in a notification making a small gif preferable. 

Is there any way the output resolution could be configurable like how we have parameters for snapshot jpegs?

**Reply by NickM-27** [reply]
There is a quality setting for the previews. You could also use the review preview endpoint which has an option to be sent as an mp4 instead and still work as the video instead of a gif
